### DISCUSSION TITLE
Skipping values while using responsive style props

### DISCUSSION THREAD

**Discussion by spidertwin2** [chakra-ui/chakra-ui#4178] [Q&A] [answered]
Hello,

I am using custom breakpoints with responsive style props.

`Box` has 4 colors `bg={["tomato", "green", "blue", "yellow"]}` but green is never applied.

Sandbox [link](https://codesandbox.io/s/kind-thunder-khlbw?file=/src/index.js).

:t-rex:

**Comment by bjcarlson42** [comment]
You can use object style syntax `bg={{sm: "tomato", md: "green", lg: "blue", xl: "yellow"}}`. View the [docs](https://chakra-ui.com/docs/features/responsive-styles#the-object-syntax).

**Reply by spidertwin2** [reply]
yes, but why does array syntax not work?

**Comment by anubra266** [comment]
The array syntax starts from `base` not `sm`. try `bg={[null, "tomato", "green", "blue", "yellow"]}`

**Reply by spidertwin2** [reply]
This skips `tomato` and only applies `null, green, blue, yellow`.

It is strange that the second array value is skipped with custom breakpoints.

**Reply by anubra266** [reply]
I've confirmed this is indeed happening wrong. Please file an issue.

**Reply by spidertwin2** [reply]
Just did #4192

**Comment by spidertwin2** [comment, marked as answer]
For anybody still facing this issue, a workaround is converting your custom breakpoints in `theme.ts` from `px` to `em/rem`.  It works then!

See https://github.com/chakra-ui/chakra-ui/issues/4192#issuecomment-869123764

**Reply by vlack-coder** [reply]
It also works using object like syntax. Thanks for the fix though... Chakra UI is a lifesaver and underrated

**Reply by Dunya-8a** [reply]
I have the same issue, it's still unfixed. Unclear why px values aren't working...
